lua/ulx/modules/sh/cfc_owoify.lua

```diff
@@ -0,0 +1,44 @@
+local CATEGORY_TYPE = "Chat"
+local OWOIFYDICT = { ove = "uv", na = "nya", ne = "nye", ni = "nyi", no = "nyo", nu = "nyu", r = "w", l = "w", O = "owo", U = "uwu", wh = "w" }
+local owoifyTargets = {}
```

Comment:
This is a fine solution, but there's one gotcha; this won't clean up players that disconnect, so as the server uptime gets bigger, so will this table.

You could watch for players who disconnect and remove them from the table and so forth, but there's actually an even easier solution.
Because this is running serverside, you can actually just plop something on the Player themselves.

So instead of doing
```lua
table.Add( owofiyTargets, ply )
```

You can just do:
```
ply.is_owoified = true
```

Then later you can just read that variable, and it'll just disappear when the player disconnects.

One more note, even if you stuck with the table approach, you could turn it into a lookup table.
So instead of:
```lua
table.Add( owoifyTargets, ply )
```
You can do:
```
owoifyTargets[ply] = true
```

Then later in your `PlayerSay` function,
```lua
if not owoifyTargets[ply] then return end
```

That's not only much faster (`table.HasValue` has to loop through the entire table), but a lot easier to work with.